certif/app/styles/components/login-form.scss

```diff
@@ -3,11 +3,13 @@
   flex-direction: column;
   margin: auto;
   align-items: center;
-  background-color: white;
+  background-color: $white;
   border-radius: 10px;
-  padding: 40px 60px;
-  max-width: 400px;
-  width: 100%;
+  padding: 20px 8px;
+
+  @include device-is('tablet') {
```

Comment:
Question pour ma compréhension, c'est du mobile first et donc on ajoute que le cas ou c'est une tablette ? Merci ^^

Reply:
exactement :D

Reply:
d'ailleurs `device-is('tablet')` inclue aussi les écrans plus grand qu'une tablette (mais s'il y avait besoin d'adapter les grands écran autrement on aurait mis derrière le devise-is('desktop'))

Reply:
pour info les breakpoints sont ici : `certif/app/styles/globals/breakpoints.scss`

Reply:
merci !